bahmni-proxy/resources/bahmni-abdm-proxy.conf

```diff
@@ -2,6 +2,6 @@ LoadModule proxy_module modules/mod_proxy.so
 LoadModule proxy_http_module modules/mod_proxy_http.so
 LoadModule rewrite_module modules/mod_rewrite.so
 
-# ndhm
-ProxyPass /ndhm http://bahmni-web:8091/ndhm
-ProxyPassReverse /ndhm http://bahmni-web:8091/ndhm
+# abha-verification
+ProxyPass /abha-verification http://abha-verification/abha-verification
+ProxyPassReverse /abha-verification http://abha-verification/abha-verification
```

Comment:
I have this comment [here](https://github.com/BahmniIndiaDistro/ABHA-Verification/pull/45/files#diff-7afe576323b0e46e79ffc67c60b6405c09c0c106467dc7758f3e08669c2cc01e)... if you let the assets sit in htdocs - you wouldnt need to duplicate `abha-verification` in `http://abha-verification/abha-verification`

Reply:
The sub-directory is needed because when the application is hit with URL `localhost/abha-verification` proxy container redirects it to `http://abha-verification/abha-verification`.  Without this the URLs at the next level are giving 404.